### PR TITLE
Set up associations

### PR DESCRIPTION
**Pull request by nicupop729**
## This feature branch holds the following:

- Create a class Classroom;
- Create the has-many/belongs-to relationship between Classroom and Student;
- Create a class Book;
- Create a class Rental;
- Create the many-to-many relationship between Person and Book using the intermediate class Rental.

### CODE REVIEW

File: rental.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```
Comment (julie-ify) [inline rental.rb]: - [ ] Let's add `:book and :person` to `attr_accessor` as users can write and read the them.

File: book.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```
Comment (julie-ify) [inline book.rb]: - [ ] Let's add `:rentals` to `attr_accessor` as users can write and read the rentals. See suggestion below:

```suggestion
  attr_accessor :title, :author, :rentals
```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @nicupop729,

Good job so far!

## Highlight!
- Descriptive README ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: book.rb (hunk 2 above)
Comment (rloterh) [reply book.rb]: This issue is well resolved! ✔️

File: rental.rb (hunk 1 above)
Comment (rloterh) [reply rental.rb]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Nicu, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**